common/test/db_cache/bok_choy_schema_student_module_history.sql

```diff
@@ -34,7 +18,7 @@ CREATE TABLE `django_migrations` (
   `name` varchar(255) NOT NULL,
   `applied` datetime(6) NOT NULL,
   PRIMARY KEY (`id`)
-) ENGINE=InnoDB AUTO_INCREMENT=119 DEFAULT CHARSET=utf8;
+) ENGINE=InnoDB AUTO_INCREMENT=88 DEFAULT CHARSET=utf8;
```

Comment:
@bradenmacdonald We do, since we remove a model entirely and move it into another app.